src/PagedListResponse.php

```diff
@@ -43,12 +44,42 @@
  * are required.
  *
  * The list elements can be accessed in the following ways:
- *  - As a single iterable using the iterateAllElements method
+ *  - As a single iterable used in a foreach loop or via the getIterator method
  *  - As pages of elements, using the getPage and iteratePages methods
  *  - As fixed size collections of elements, using the
  *    getFixedSizeCollection and iterateFixedSizeCollections methods
+ *
+ * Example of using PagedListResponse as an iterator:
+ * ```
+ * $pagedListResponse = $client->getList(...);
+ * foreach ($pagedListResponse as $element) {
+ *   // doSomethingWith($element);
```

Comment:
Could we please use 4 spaces?

Reply:
agreed, these are great but let's use 4 spaces!!

Reply:
Done!